fntypes.proto

```diff
@@ -14,21 +14,14 @@
  * limitations under the License.
  */
 syntax = "proto3";
-import "google/protobuf/empty.proto";
 
 package fntypes;
 
-message Request {
-    string body = 1;
-}
-
-/*
- * TODO: Reply can be Empty for sink behavior. Maybe we should create a separate RPC for sink.
- * For now the Dispatcher can check the return type and decide to output the Reply.
- */
-message Reply {
-    oneof type {
-	    string body = 1;
-	    google.protobuf.Empty void = 2;
+message Message {
+	message HeaderValue {
+		repeated string values = 1;
```

Comment:
why `repeated` here?

Reply:
Because we want a `MultiMap<String, String>`

Reply:
@ericbottard decided we should represent multi-value headers explicitly (like HTTP)

Reply:
LGTM;